components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/exception/ActionMgtServerException.java

```diff
@@ -23,29 +23,18 @@
  */
 public class ActionMgtServerException extends ActionMgtException {
 
-    public ActionMgtServerException(String message, String errorCode) {
-
-        super(message, errorCode);
-    }
-
-    public ActionMgtServerException(String message, String description, String errorCode) {
+    public ActionMgtServerException(String message) {
```

Comment:
Check if this function is really needed

Reply:
We have one usage for this. Hence keeping it as it is